src/main/java/org/xbill/DNS/LookupResult.java

```diff
@@ -0,0 +1,61 @@
+package org.xbill.DNS;
+
+import static java.util.Objects.hash;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import org.xbill.DNS.exceptions.AdditionalDetail;
+
+/** LookupResult instances holds the result of a successful lookup operation. */
+public class LookupResult {
```

Comment:
If the intention is that this class is immutable, make it `final`.

Reply:
Good catch, updated